websocket.go

```diff
@@ -220,18 +223,18 @@ func (c *Client) Send(dst *network.ServerIdentity, path string, buf []byte) ([]b
 		if err != nil {
 			return nil, NewClientError(err)
 		}
-		c.si = dst
+		c.connections[dest] = conn
 	}
 	defer func() {
 		if !c.keep {
-			c.Close()
+			c.Close(dest)
```

Comment:
Isn't this done [here](https://github.com/dedis/onet/pull/194/files/791fa463220631c03ba26654e1b8bb1f250bdf3d#diff-8990d2c9c7f1690e60641d425c8932f7R294)?